Makefile

```diff
@@ -10,6 +10,9 @@ TESTS = $(wildcard test/*.py)
 serve:
 	export PROJECTPATH=${PROJECTPATH} && gulp prod-js && cd ./src && python3 server.py
```

Comment:
Just want to emphasize why gulp is here. Right now, we don't share a common version of all.js which the HTML template points to. This makes sure that the react code is ready to go whenever the server is started.